tests/mbt/model/ccv_pss_test.qnt

```diff
@@ -0,0 +1,34 @@
+// This module contains logic for PSS (Partial Set Security).
+// PSS is a variant/extension of CCV that 
+// allows for only a subset of the validator set
+// to secure a consumer chain.
+// Not all logic related to PSS is inside this module, as some logic is
+// too tightly coupled with the core CCV logic,
+// which is instead found in ccv.qnt
+module ccv_pss_test {
+    import ccv_types.* from "./ccv"
+    import extraSpells.* from "./libraries/extraSpells"
+    import ccv_utils.* from "./ccv_utils"
+    import ccv_pss.* from "./ccv_pss"
+
+    run TopNTest = 
+        val valSet =
+            Map("a" -> 5, "a1" -> 5, "a2" -> 5, "b" -> 10, "b1" -> 10, "b2" -> 10, "c" -> 15, "c1" -> 15, "e" -> 25)
```

Comment:
nit: Why `e` is the last one and not `d`?

Reply:
d was rationalized away at some point and I didn't change this :D
yeah, sorting the map the other way around is also a great point